### PR TITLE
Add default site to admin users

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@codiebeulaine @miltontony ready for review :)

File: molo/profiles/models.py [hunk 1]
```diff
@@ -331,6 +331,7 @@ class UserProfile(models.Model):
 def user_profile_handler(sender, instance, created, **kwargs):
     if created:
         profile = UserProfile(user=instance)
+        profile.site = Site.objects.get(is_default_site=True)
```
Comment (miltontony) [inline molo/profiles/models.py:334]: I dont think this will work.. lets chat at 2

**Comment by smarzban** [comment]
@miltontony @codiebeulaine ready for review

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**